bin/storm.py

```diff
@@ -241,7 +240,6 @@ def jar(jarfile, klass, *args):
             extrajars=[tmpjar, USER_CONF_DIR, STORM_BIN_DIR],
             args=args,
             daemon=False,
```

Comment:
The goal was so that the code after this call would execute.
```
os.remove(tmpjar)
```

Without that we leak a file.